modules/conversion-center/module.php

```diff
@@ -96,6 +105,50 @@ public function __construct() {
 		add_action( 'wp_ajax_elementor_send_clicks', [ $this, 'handle_click_tracking' ] );
 		add_action( 'wp_ajax_nopriv_elementor_send_clicks', [ $this, 'handle_click_tracking' ] );
 
+		if ( ! static::is_pro_active() ) {
+			add_action( 'wp_footer', function () {
```

Comment:
@nicoladj77

Can you please paste the output of the HTML here? I just want to make sure of something.